books.js

```diff
@@ -2,63 +2,76 @@ const titleObj = document.querySelector('#title');
-    id: Math.floor(Math.random() * 10000),
-  };
-  totalBooks.push(bookObj);
-  localStorage.setItem('store', JSON.stringify(totalBooks));
-}
+  // eslint-disable-next-line class-methods-use-this
+  displayBooks(bookObj) {
+    const holder = document.createElement('div');
+    const titleAuthor = document.createElement('p');
+    const removeBtn = document.createElement('button');
 
-// Function to remove from the local storage using created ID
+    titleAuthor.innerText = `"${bookObj.title}" by ${bookObj.author}`;
+    removeBtn.innerText = 'Remove';
+    removeBtn.setAttribute('data-id', bookObj.id);
+    removeBtn.className = 'btn';
+    titleAuthor.className = 'title-author';
+    holder.className = 'book-holder';
+    holder.append(titleAuthor);
+    holder.append(removeBtn);
+    container.append(holder);
+  }
 
-function bookRemove(id) {
-  totalBooks = totalBooks.filter((container) => container.id !== id);
-  localStorage.setItem('store', JSON.stringify(totalBooks));
+  // eslint-disable-next-line class-methods-use-this
+  deleteBook(e) {
+    if (e.classList.contains('btn')) {
+      // Retrieve data-id attribute.
+      // It's a string, change it into number
+      const id = Number(e.getAttribute('data-id'));
+      Books.totalBooks = Books.totalBooks.filter((book) => book.id !== id);
+      localStorage.setItem('store', JSON.stringify(Books.totalBooks));
+      e.parentElement.remove();
+      if (Books.totalBooks.length === 0) {
+        container.innerHTML = '<p class="no-books">No books available!</p>';
+      }
+    }
+  }
 }
 
-// Creating Dynamic BOOKS
-
-function createBook(bookObj) {
-  const holder = document.createElement('div');
-  const bkTitle = document.createElement('h3');
-  const bkAuthor = document.createElement('p');
-  const removeBtn = document.createElement('button');
-  const line = document.createElement('hr');
-
-  // Assigning Innertext to Created elements
-
-  bkTitle.innerText = bookObj.title;
-  bkAuthor.innerText = bookObj.author;
-  removeBtn.innerText = 'Remove Book';
-  holder.append(bkTitle);
-  holder.append(bkAuthor);
-  holder.append(removeBtn);
-  holder.append(line);
-  container.append(holder);
+const book = new Books();
 
-  // Event listener for the button
-
-  removeBtn.addEventListener('click', () => {
-    removeBtn.parentElement.remove();
-    bookRemove(bookObj.id);
+function loopFunc() {
+  Books.totalBooks.forEach((e) => {
+    book.displayBooks(e);
   });
 }
-
-totalBooks.forEach(createBook);
+loopFunc();
 
 formObj.addEventListener('submit', (e) => {
   e.preventDefault();
-  if (titleObj.value !== '' || authorObj.value !== '') {
-    addBooks();
-    createBook(bookObj);
-    formObj.reset();
-  }
+  const id = Math.floor(Math.random() * 10000);
+  const book = new Books(titleObj.value, authorObj.value, id);
+  container.innerHTML = '';
+  Books.totalBooks.push(book);
+  localStorage.setItem('store', JSON.stringify(Books.totalBooks));
+  loopFunc();
+  formObj.reset();
```

Comment:
Kindly create an `add book` method inside the class and move the functionality for adding a new book to the `add book` method inside the class as per project requirements.

![image](https://user-images.githubusercontent.com/38088818/191741015-0e88a31e-62f9-4f92-b005-29317e5565b0.png)

- I notice that you are using the class instance `const book = new Books()` to create a new book. The are other ways to achieve this without relying on class instances for creating each new book.

Kindly consider the example below:

![image](https://user-images.githubusercontent.com/38088818/191610953-36dd1d41-a314-43c4-bec3-6883c24f0379.png)